### PR TITLE
Landsat-util

### PR DESCRIPTION
**Pull request by ocefpaf**
@bucricket this is the first attempt to package `landsat-util`.

xref: https://github.com/conda-forge/gdal-feedstock/issues/146

Note that following the exact pinning from `requirement.txt` file is quite complicated b/c conda-forge dropped `numpy 1.10` and many packages would need to be rebuilt with an unsupported version. I opened an issue to track that at https://github.com/developmentseed/landsat-util/issues/223

PS: I am running the tests in this recipe and there are some failures that need some investigation. I do not have time for that right now, but I appreciate if you or someone more familiar with `lansat-util` could take a look.

### CODE REVIEW

**Comment by bucricket** [comment]
What are the failures?  Sorry I’m really new to this process.

**Comment by ocefpaf** [comment]
> What are the failures?  Sorry I’m really new to this process.

@bucricket there are many tests that are failing and I do not believe they are related to the version of `rasterio` or `gdal`. (`geocoder` maybe?)

```shell
            else:
                result['status'] = u'SUCCESS'
>               result['total'] = r_dict['meta']['results']['total']
E               KeyError: 'results'

../_t_env/lib/python3.5/site-packages/landsat/search.py:141: KeyError
____________________________ TestUtils.test_geocode ____________________________

self = <tests.test_utils.TestUtils testMethod=test_geocode>

    def test_geocode(self):
        loc = utils.geocode('1600 Pennsylvania Ave NW, Washington, DC 20500')
    
        self.assertEqual(round(loc['lat'], 3), 38.898)
>       self.assertEqual(round(loc['lon'], 3), -77.037)
E       AssertionError: -77.036 != -77.037
```

See the full log at: https://circleci.com/gh/ocefpaf/staged-recipes/515

**Comment by bucricket** [comment]
Ok I think we should get the author of Landsat-util involved then.

Mitch

**Comment by Juanlu001** [comment]
The requirements are ready :)

File: recipes/landsat-util/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - tests
+  requires:
+    - pytest
+    - mock
+    - jsonschema
+  imports:
+    - landsat
+  commands:
+    - landsat --help
+    - py.test -s -rxs -v -k "not (test_search_json_output or test_search_lat_lon or test_search_pr_correct  or test_search_pr_wrong_input or test_too_many_results or test_search or test_search_with_geojson or test_search_zero_lon or test_geocode)" tests
```
Comment (ocefpaf) [inline recipes/landsat-util/meta.yaml]: It would be nice to have a py.test maker to skip the tests that require the API key.

**Comment by Juanlu001** [comment]
I think this is more than enough. I kept wide intervals on purpose, but as you say some versions are too old. Thanks for pushing this forward :)

**Pull request merged**